commit 565988ab47bd9b96b50608564aee2104aeb4b7ae
Author: Felix Fietkau <nbd@nbd.name>
Date:   Tue Dec 13 14:20:49 2016 +0100

    gcc: rip out transactional memory related bloat from crtbegin
    
    Slightly improves compression for each executable, saving about 4k from
    the default ar71xx rootfs
    
    Signed-off-by: Felix Fietkau <nbd@nbd.name>

---
 libgcc/crtstuff.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/libgcc/crtstuff.c
+++ b/libgcc/crtstuff.c
@@ -155,7 +155,7 @@
 #endif
 
 #if !defined(USE_TM_CLONE_REGISTRY) && defined(OBJECT_FORMAT_ELF)
-# define USE_TM_CLONE_REGISTRY 1
+# define USE_TM_CLONE_REGISTRY 0
 #endif
 
 /* We do not want to add the weak attribute to the declarations of these
